screens/RecipeListScreen.js

```diff
@@ -16,8 +16,8 @@ import styles from "./styles.js";
 // Recipe List Component
 const RecipeListScreen = (props) => {
   return (
-    <KeyboardAwareScrollView>
-      <SafeAreaView style={styles.containerRecipeList}>
+    <KeyboardAwareScrollView style={styles.containerRecipeList}>
```

Comment:
Was there a reason why the position of this code was changed?

Reply:
since safeAreaView is nested, the background color was being cutoff at the top, making it blue only where the recipelist finder and card are; instead of the entire device screen.